tool/gravity/cli/reconfigure.go

```diff
@@ -150,3 +139,33 @@ func newReconfigurator(ctx context.Context, config *install.Config, state *recon
 	}
 	return installer, nil
 }
+
+// validateReconfiguration determines if reconfiguration can be launched on this
+// node.
+//
+// If all checks pass, returns information about the existing cluster state such
+// as the cluster object and the original install operation.
+func validateReconfiguration(env *localenv.LocalEnvironment, config InstallConfig) (*reconfigure.State, error) {
+	// The cluster should be installed but not running.
+	err := localenv.DetectCluster(env)
+	if err != nil && trace.IsNotFound(err) {
+		return nil, trace.BadParameter("Cluster doesn't appear to be installed on this node.")
```

Comment:
nit: `Cluster` and `node` don't mix well when reading this - how about `Gravity is not installed`?